static/test_io_close.py

```
@@ -0,0 +1,22 @@
+
+
+def func1():
+    # 检测出
+    fd = open('test.txt')
+    results = fd.read()
+    return results
+
+
+def func2():
+    # 未检测
+    fd = open('test.txt')
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)